File: conceptos/loops.py
# Loops
""" 

for index in collection
    codigo


while condition:
    codigo

range(start = 1, stop = 11, step = 1) # start = 0, step = 1
    

"""
numeros = [1, 2, 3, 4, 5, 6]

for i in range(1, 11):
    print(i)

for i in range(len(numeros)):
    print(numeros[i])

for numero in numeros:
    print(numero)


indice = 0
while indice < len(numeros):
    print(numeros[indice])
    indice+=1 
    #indice = indice + 1 

File: conceptos/loops.js
/* Loops */
/* 

for(iterator; condition; increment){
    codigo
}

for(index in collection){
    codigo
}

for(valor of collection){
    codigo
}

while(condition){
    codigo
}

do {
    codigo
} while (condition)


*/

let numeros = [1, 2, 3, 4, 5, 6]

for(let i = 1; i <= 10; i++){
    console.log(i)
}

for(let i = 0; i < numeros.length; i++ ){
    console.log(numeros[i])
}

for(let indice in numeros){
    console.log(numeros[indice])
}

for(let valor in numeros){
    console.log(valor)
}

let indice = 0;
while(indice < numeros.length){
    console.log(numeros[indice])
    indice++
}

indice = 0;
do {
    console.log(numeros[indice])
    indice++
} while (indice < numeros.length)
